test/cmd/projects.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+OS_ROOT=$(dirname "${BASH_SOURCE}")/../..
+source "${OS_ROOT}/hack/lib/init.sh"
+os::log::stacktrace::install
+trap os::test::junit::reconcile_output EXIT
+
+os::test::junit::declare_suite_start "cmd/projects"
+os::cmd::expect_failure_and_text 'oc projects test_arg' 'no arguments'
+os::cmd::expect_success_and_text 'oc projects' 'You have access'
```

Comment:
Don't use this test -- see https://github.com/openshift/origin/pull/10047#discussion_r72458000